pkg/proxy/cache.go

```diff
@@ -6,73 +6,62 @@ import (
 
 	"github.com/codeready-toolchain/registration-service/pkg/application"
 	"github.com/codeready-toolchain/registration-service/pkg/log"
-	"github.com/codeready-toolchain/registration-service/pkg/proxy/namespace"
+	"github.com/codeready-toolchain/registration-service/pkg/proxy/access"
 
 	"github.com/gin-gonic/gin"
 )
 
-type UserNamespaces struct {
+type UserAccess struct {
 	sync.RWMutex
 	app application.Application
 
-	// namespace accesses by username
-	namespaces map[string]*namespace.NamespaceAccess
+	// cluster accesses by username
+	clusterAccesses map[string]*access.ClusterAccess
 }
 
-func NewUserNamespaces(app application.Application) *UserNamespaces {
-	return &UserNamespaces{
-		app:        app,
-		namespaces: make(map[string]*namespace.NamespaceAccess),
+func NewUserAccess(app application.Application) *UserAccess {
+	return &UserAccess{
+		app:             app,
+		clusterAccesses: make(map[string]*access.ClusterAccess),
 	}
 }
 
-// Get tries to retrieve the namespace access from the cache first. If found then it checks if the cached access is still valid.
-// If not found or invalid then retrieves a new namespace access from the member cluster service and stores it in the cache.
-func (c *UserNamespaces) Get(ctx *gin.Context, userID, username string) (*namespace.NamespaceAccess, error) {
-	na, err := c.namespaceFromCache(ctx, username)
-	if err != nil {
-		return nil, err
-	}
-	if na != nil {
-		return na, nil
+// Get tries to retrieve the cluster access from the cache first. If found then it checks if the cached access is still valid.
```

Comment:
It doesn't validate the access anymore. Let's fix the comment.

Reply:
Good catch, done in d36401dd79ba7c2e9467cd07278381699783fd21